nservicebus/recoverability/index.md

```diff
@@ -0,0 +1,112 @@
+
+## Delayed Retries
+
+Delayed Retries introduces another level of retry mechanism for messages that fail processing. Delayed Retries picks up the message and defers its delivery, by default first for 10 seconds, then 20, and lastly for 30 seconds, then returns it to the original worker queue.
+
+For example, if there is a call to an web service in the handler, but the service goes down for five seconds just at that time. Without Delayed Retries, the message is retried instantly and sent to the error queue. With Delayed Retries, the message is instantly retried, deferred for 10 seconds, and then retried again. This way, when the Web Service is available the message is processed just fine.
+
+For more information about how to configure delayed retries, refer to [configure delayed retries](/nservicebus/recoverability/configure-delayed-retries.md).
+
+NOTE: Retrying messages for extended periods of time would hide failures from operators, thus preventing them from taking manual action to honor their Service Level Agreements. To avoid this, NServiceBus will make sure that no message is retried for more than 24 hours before being sent the error queue.
+
+### Transport transaction requirements
+
+The Delayed Retries mechanism is implemented by rolling back the [transport transaction](/nservicebus/transports/transactions.md) and scheduling the message for [delayed-delivery](/nservicebus/messaging/delayed-delivery.md). Aborting the receive operation when transactions are turned off would result in a message loss. Therefore Delayed Retries cannot be used when transport transactions are disabled and delayed-delivery is not supported.
+
+## Fault handling
+
+When messages failed during the immediate retries and delayed retries mechanism they will be moved to the error queue. Fault handling requires a configured error queue. For more information how to configure the error queue refer to [configure error queue](/nservicebus/recoverability/configure-error-queue.md).
+
+### Transport transaction requirements
+
+Fault handling doesn't require to roll back the transport transaction. A copy of the currently handled message is sent to the configured error queue and the current transaction will be marked as successfully processed. Therefore fault handling works with all supported [transport transaction modes](/nservicebus/transports/transactions.md).
+
+## Recoverability Policy
+
+NServiceBus Version 5 and lower allowed to take full control over the delayed retries part of Recoverability only by providing a [custom delayed retries policy](/nservicebus/recoverability/configure-delayed-retries.md).
+
+Starting from Version 6 it is possible to take full control over the whole Recoverability process. For more information refer to [custom recoverability policy](/nservicebus/recoverability/custom-recoverability-policy.md). 
+
+## Total number of possible retries
```

Comment:
This is valid for versions up to 5. With V6 `MaxRetires` does what it actually says. We need to update this section.